### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#1] [Feature request]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * [Share feature request](https://github.com/open-sauced/feedback/discussions/categories/feature-request).
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * [Request repositories for indexing on OpenSauced](https://github.com/open-sauced/feedback/discussions/2)

  To get started, comment below with an introduction of yourself and tell us about what you would like to get out of this community.

**Comment by shubham-singh-748** [comment]
Hello there myself Shubham from India. I liked the most about OpenSauced which is a reason I'm here that OpenSauced has a very beginner friendly environment. Like, i thought of open source contribution, but i got so many questions like how to get started, how to find active projects, etc...
But here i find all solutions to my problems. I will recommend OpenSauced to my friend who want to start open source contribution but confused where to start.

**Reply by sohaibchebah** [reply]
have a good joureny shubham 😊

**Reply by BekahHW** [reply]
We just shipped a new feature called the OSCR that could help you track your progress. You can read more about it here: https://opensauced.pizza/blog/numbers-and-narratives.

Also, check out our #100DaysOfOSS Challenge - #36

**Comment by StephenMuya** [comment]
Hello.
My name is Stephen, and I am from Nairobi, Kenya. I am a CS student at the University of the People, and I am looking forward to making impactful contributions here.

**Reply by adiati98** [reply]
Welcome, @StephenMuya! 👋🏼

**Reply by StephenMuya** [reply]
Thank you @adiati98

**Comment by Cade-Bray** [comment]
Hey!
My name is Cade and I'm finishing up my CS degree. I've also worked two years as a software developer. I'm really trying to break into open source for the skill building effect of it. I want new and challenging problems that are not mindless leetcode problems. I've started following the Intro course GitHub repo that OpenSauced maintains. I am still finding it difficult to find open source projects on the Open-Sauced site that are not informational repositories, like Microsoft's beginner ML course.

**Comment by Anilkollu** [comment]
Hello All,
I am a graduated student from Boston, MA and am very much interested for contributing opensource projects, can anyone please guide me to do so? I am new to OpenSauced.

**Comment by samuel-sysnix** [comment]
Hello everyone, my name is Samuel. I’m currently pursuing a degree in Information Security at Unifatecie College here in Brazil, and I’m part of the penguin (GNU/Linux) community. I’d appreciate suggestions for practical training, literature, and free certifications in my field.

**Comment by vivekd16** [comment]
Hi everyone! 👋 I'm Vivek Deshmukh, a Data Science student passionate about AI, ML, and open-source development. I'm currently learning and building projects around machine learning. Excited to be a part of the OpenSauced community and looking forward to learning, collaborating, and contributing together! 🚀
GitHub: [github.com/vivekd16](https://github.com/vivekd16)
LinkedIn: [linkedin.com/in/vivek-deshmukh-66845822b](https://www.linkedin.com/in/vivek-deshmukh-66845822b)

**Comment by rishi-jat** [comment]
Hey everyone! 👋
I’m Rishi, new to open source and really excited to start my journey here.
I love learning new things and contributing to real-world projects. Looking forward to connecting with you all and growing together! 🚀

**Comment by SaifRasool92** [comment]
Hi I am Saif. I am a student in a college and I am also a hackathon and coding competitions participant and i love to teach and learn from others. I am a part of a cummunity where we do everything volunteerly which include teaching DSA, Python, AI and making underpriveledged students ready for international career. I am here to learn from you all GUYS! I would love to discuss ideas, share knowledge and much more...!
I really love to connect with people so please connect with me on linkedin: https://www.linkedin.com/in/saif-ur-rasool/
